ntoskrnl/mm/ARM3/procsup.c

```diff
@@ -1416,8 +1421,7 @@ MmDeleteProcessAddressSpace(IN PEPROCESS Process)
     }
     else
     {
-        /* A partly-initialized process should never exit through here */
-        ASSERT(FALSE);
+        DPRINT1("Deleting partially initialized address space. Might leak resources.");
```

Comment:
Missing `\n`. Could also print some info about the process (EPROCESS pointer or PID maybe)